docs/Concepts/PreparingForTheMerge.md

```diff
@@ -0,0 +1,63 @@
+
+Once that difficulty is reached, eth1 will then become the execution layer of Ethereum, and the consensus layer will be responsible for progressing the chain.
+
+The consensus layer will enable Bellatrix (the merge configuration) prior to the terminal difficulty being reached, then it is up to eth1 blocks to reach the pre-determined difficulty, then from that point Ethereum moves to a proof of stake network.
+
+Post merge, as well as validators earning rewards for performing duties, there are also earnings from the execution engine performing work, and these earnings are directed to a fee recipient address.
+
+## Execution Engine
+
+Pre-merge, an eth1-client is required for block proposal to get deposits for validators. They're intermittent, and you get the data from other blocks if your eth1 client is offline for some reason.
+
+Once the merge is complete, the execution engine becomes much more heavily relied upon for execution. It will be no longer viable for an external provider to be referenced, each beacon-node will need to be closely associated with an execution engine (eth1 client).
+
+Each beacon-node will require its own execution-engine, so if you are maintaining multiple beacon-nodes, you will need to set up an execution-engine for each beacon-node. Service providers that provide eth1 access will not be adequate for a beacon-node to continue to function on the network.
+
+To configure the execution-engine for teku, the [ee-endpoint](https://docs.teku.consensys.net/en/latest/Reference/CLI/CLI-Syntax/#ee-endpoint) will need to be set in teku configuration. this can be done prior to the merge block, and can replace [eth1-endpoint](https://docs.teku.consensys.net/en/latest/Reference/CLI/CLI-Syntax/#ee-endpoint).
+
+JWT authentication is used to secure the communication between beacon-node and execution engine. This can be generated through the use of a command line tool, for example:
+```
+openssl rand -hex 32 -out <file>
+```
+
+The JWT token will then be provided to teku with [ee-jwt-secret-file](https://docs.teku.consensys.net/en/latest/Reference/CLI/CLI-Syntax/#ee-jwt-secret-file), and also provided to the chosen execution-endpoint through its configuration options.
+
+Teku configuration changes will look similar to:
+```
+< eth1-endpoint: "http://eth1endpoint:8545/"
+---
+> ee-endpoint: "http://localhost:8550/"
+> ee-jwt-secret-file: "secret.hex"
+```
+
+The execution-endpoint requirements depends on the client used, and there is no requirement to use any particular eth1 client. The traffic between execution endpoint and the  beacon-node will be relatively high, so that should be taken into account when planning your solution.
+
+Prior to Bellatrix coming online, all eth1 clients required for use as an execution-engine should be sync'd on mainnet so that this does not delay network participation. Validators are not able to produce blocks without an execution-endpoint that is fully synchronized.
```

Comment:
```suggestion
Prior to Bellatrix coming online, all eth1 clients required for use as an execution-engine should be sync'd on mainnet so that this does not delay network participation. Validators are not able to produce attestations or blocks without an execution-endpoint that is fully synchronized.
```

The beacon node will remain in "Syncing" mode until the execution client has completed it's sync and no validator duties will be performed.